wp-content/themes/core/components/accordion/accordion.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$controller = \Tribe\Project\Templates\Components\accordion\Controller::factory( $args );
+?>
+<div
+	<?php echo $controller->container_classes(); ?>
+		role="tablist"
+		aria-multiselectable="true"
+	<?php echo $controller->container_attrs(); ?>
+>
+	<?php foreach ( $controller->rows as $key => $row ) { ?>
+	<article <?php echo $controller->row_classes(); ?>>
+		<<?php echo esc_attr( $controller->row_header_tag ); ?>>
+		<button
```

Comment:
Similar to the above comment, I wonder if we setup all these attributes in the controller instead of mixing them here? Alternatively, can the dynamic (classes output) to the first or last attribute and the white space aligned? (This applies below as well.)